topsail/_common.py

```diff
@@ -162,6 +162,23 @@ def _run(self):
         print(f"Using '{env['ANSIBLE_CACHE_PLUGIN_CONNECTION']}' to store ansible facts.")
         Path(env["ANSIBLE_CACHE_PLUGIN_CONNECTION"]).parent.mkdir(parents=True, exist_ok=True)
 
+        # We configure the roles path dynamically appending them to the defaults
+        top_dir_roles = Path(top_dir / "roles")
+        # top_dir_roles_list = [os.path.join(top_dir_roles, entry) for entry in os.listdir(top_dir_roles) if os.path.isdir(os.path.join(top_dir_roles, entry))]
+        top_dir_roles_list = [str(entry) for entry in top_dir_roles.glob('*') if entry.is_dir()]
+        current_roles_path = env.get("ANSIBLE_ROLES_PATH", "")
+        env["ANSIBLE_ROLES_PATH"] = os.pathsep.join([current_roles_path] + top_dir_roles_list)
+        self.ansible_vars["roles_path"] = env["ANSIBLE_ROLES_PATH"]
+
+        # We configure the collections path dynamically
+        current_collections_paths = [env.get("ANSIBLE_COLLECTIONS_PATHS", "tikitiki")]
+        for path in sys.path:
+            collections_path = str(Path(path) / 'ansible_collections')
+            if os.path.exists(collections_path):
+                current_collections_paths = current_collections_paths + [collections_path]
```

Comment:
```
current_collections_paths = current_collections_paths + [collections_path]
```
this form ^^^ should be avoided,  this one vvv is prefered:
```
current_collections_paths += [collections_path]
```
and this ^^^ and this vvv is equivalent:
```
current_collections_paths.append(collections_path)
```
I use one or the other depending on the wind, not sure if there's any strong argument to prefer one side over the other